src/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Auto-Magazine</title>
+</head>
+
+<body class="page">
+  <header class="menu">
+    <nav class="menu__nav">
+      <ul class="menu__nav-list">
+        <li class="menu__nav-item"><a class="menu__nav-link" href="#">ФАКТЫ</a></li>
+        <li class="menu__nav-item"><a class="menu__nav-link" href="#">ГАЛЕРЕЯ</a></li>
+        <li class="menu__nav-item"><a class="menu__nav-link" href="#">ПРЕССА</a></li>
+      </ul>
+    </nav>
+    <a href="#" class="logo logo_type_header"></a>
+    <button class="burger active-el" type="button" aria-pressed="false"></button>
+  </header>
+  <main class="сontent">
+    <section class="slide">
+      <p class="slide__brand">volkswagen T1</p>
+      <h1 class="slide__title">Новая версия популярного микроавтобуса</h1>
+      <h2 class="slide__subtitle">С ЭЛЕКТРО ДВИГАТЕЛЕМ</h2>
```

Comment:
Всё же мне кажется лучше будет сделать так:
```suggestion
      <h1 class="slide__brand">volkswagen T1</h1>
      <p class="slide__title">Новая версия популярного микроавтобуса</p>
      <p class="slide__subtitle">С ЭЛЕКТРО ДВИГАТЕЛЕМ</p>
```

Т.е. название машины в `h1`, а описание в `p`.